africanus/gridding/perleypolyhedron/tests/test_ppgridder.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+import numpy as np
+from africanus.gridding.perleypolyhedron import kernels, gridder, policies
+import os
+
+DEBUG = True
+
+class griddertest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        unittest.TestCase.setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
+        unittest.TestCase.tearDownClass()
+
+    def tearDown(self):
+        unittest.TestCase.tearDown(self)
+        
+    def setUp(self):
+        unittest.TestCase.setUp(self)
+        
+    def test_construct_kernels(self):
```

Comment:
Convert to a pytest test case by removing the class and making this a standalone test function.

See https://github.com/ska-sa/codex-africanus/pull/192/files#diff-c9ac70c00217b6f0452ae38107b622feR11-R12 for an example of how I changed your bounding box test case.